lib/screen/onboarding_page.dart

```diff
@@ -57,6 +58,10 @@ class _OnboardingPageState extends State<OnboardingPage>
     handleBranchLink();
     handleDeepLink();
     handleIrlLink();
+    Future.delayed(const Duration(seconds: 1), () {
+      MigrateAddressUtil(injector(), injector(), injector())
```

Comment:
I think we can put this migration run inside the `MigrationUtil.migrateIfNeeded`.